tests/CMakeLists.txt

```diff
@@ -32,10 +32,6 @@ if (NOT LIBIA2_AARCH64 AND NOT IA2_LIBC_COMPARTMENT)
 endif()
 
 if (IA2_LIBC_COMPARTMENT)
-    if (LIBIA2_AARCH64)
-        message(FATAL_ERROR "IA2_LIBC_COMPARTMENT is not supported on AArch64")
-    endif()
-
```

Comment:
The `return()` on line 48 needs to be removed. It's not doing anything right now, but could cause cmake to do the wrong thing depending on how this CMakeLists.txt gets included.